build.gradle

```diff
@@ -14,6 +15,18 @@ repositories {
   jcenter()
 }
 
+idea {
```

Comment:
Why wouldn't you want the source and javadoc available in your projects?

Reply:
No, I just do not want, chyotba they were downloaded always, but not when it is necessary to the developer - it saves a traffic and time of project initiation. If there is a need, then in IDEA it is always possible to finish loading sourses or javadoc: https://youtrack.jetbrains.com/issue/IDEA-174225

Reply:
This seems like more of a developer preference thing than something specific to this project. I'd suggest leveraging [Gradle init scripts](https://docs.gradle.org/current/userguide/init_scripts.html) to disable the sources/javadoc for all projects on your machine.

Once you pull this piece out of the PR, I can merge it in.